src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -214,7 +214,11 @@ def _verify_job_id_list(self, req: JobRequest) -> None:
             )
             raise NoJobException("No valid job ids")
 
-    def start_job_status_loop(self, *args, **kwargs) -> None:
+    def start_job_status_loop(
+        self,
+        init_jobs: bool = False,
+        cell_list: List[str] = None,
+    ) -> None:
         """
         Starts the job status lookup loop. This runs every 10 seconds.
         This has the bare *args and **kwargs to handle the case where this comes in as a job
```

Comment:
looks like these docs need to be updated

Reply:
Oof good catch. It points out that the first positional argument also needs to be a `JobRequest`. Adding tests